jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/DateInternationalizationHelper.java

```diff
@@ -80,4 +80,8 @@ public static int getFirstDayOfWeek(Locale locale) {
         Calendar calendar = Calendar.getInstance(locale);
         return calendar.getFirstDayOfWeek();
     }
+
+    public static int calendarDayOfWeekToVaadinDayOfWeek(int calenderDayOfWeek) {
```

Comment:
We don't need a separate method, just fix `getFirstDayOfWeek`.

Reply:
I have tried to do the changes you requested. Please let me know if you have other change requests.